.github/workflows/check-pull-request.yml

```diff
@@ -161,3 +161,8 @@ jobs:
 
       - name: Run task
         run: ${{ matrix.task.run }}
+
+      - name: SonarCloud Scan
+        uses: SonarSource/sonarcloud-github-action@master
+        env:
+          SONAR_TOKEN: ${{ secrets.SONARCLOUD_TOKEN }}
```

Comment:
looks like this doesn't exist - @alexluckett

Reply:
Ah I do have one, I must not have applied it to the repo. I'll dig it out and apply it this afternoon.